modular_zubbers/code/modules/cargo/packs/goodies.dm

```diff
@@ -4,19 +4,33 @@
 	contains = list(/obj/item/gun/ballistic/automatic/pistol/sol = 1,
 	/obj/item/ammo_box/magazine/c35sol_pistol/starts_empty = 1,
 	)
-	cost = PAYCHECK_COMMAND * 20
-	access_view = ACCESS_WEAPONS //The cost of a Detective Revolver
+	cost = PAYCHECK_COMMAND * 10 //Half the cost of a Detective Revolver
+	access_view = ACCESS_WEAPONS
 
 /datum/supply_pack/goody/sol_revolver_single
 	name = "Sol 'Eland' Revolver Single Pack"
 	desc = "A stubby revolver chiefly found in the hands of Private Security forces due to its cheap price and decent stopping power. Comes with an ammo box."
 	contains = list(/obj/item/gun/ballistic/revolver/sol = 1,
 	/obj/item/ammo_box/c35sol = 1,
 	)
-	cost = PAYCHECK_COMMAND * 20 //The cost of a Detective Revolver
+	cost = PAYCHECK_COMMAND * 10 //Half the cost of a Detective Revolver
 	access_view = ACCESS_WEAPONS
 //Cargo techs when no permit
 
+/datum/supply_pack/goody/disablersmg_single
+	name = "Disabler SMG Single-Pack"
+	desc = "Contains one disabler SMG, an automatic variant of the original workhorse."
+	cost = PAYCHECK_COMMAND * 3
+	access_view = ACCESS_WEAPONS
+	contains = list(/obj/item/gun/energy/disabler/smg)
+
+/datum/supply_pack/goody/lasercarbine_single
+	name = "Energy Gun Single-Pack"
```

Comment:
If this is lethal only, it should be named "laser carbine" or "laser gun", energy guns are the dual-mode ones

Reply:
Yeah I secondthis

Reply:
Think it was just a leftover, I'll fix it